provider/cmd/pulumi-resource-cloudflare/schema.json

```diff
@@ -3183,8 +3246,7 @@
                     "description": "The ID of this resource.\n"
                 },
                 "name": {
-                    "type": "string",
-                    "description": "The name of the notification policy.\n"
+                    "type": "string"
```

Comment:
and here we also loose docs that seem correct?

Reply:
There is no upstream documentation associated with this field:
https://github.com/cloudflare/terraform-provider-cloudflare/blob/8008476152ba7f6e8d69dfc085a4dc2264c8f5b8/docs/resources/notification_policy.md#L94